### PR TITLE
[CN-1164]Added MC CR to upgrade test

### PR DESCRIPTION
**Pull request by cheels**
## Description
Added MC CR to upgrade test during release

### CODE REVIEW

File: .github/workflows/test-and-release.yaml [hunk 1]
```diff
@@ -149,12 +149,26 @@ jobs:
...
                 maxSizePolicy: USED_NATIVE_MEMORY_SIZE
                 size: 5000
+          ---
+          apiVersion: hazelcast.com/v1alpha1
+          kind: ManagementCenter
+          metadata:
+            name: managementcenter
+          spec:
+            repository: 'hazelcast/management-center'
+            externalConnectivity:
```
Comment (kutluhanmetin) [inline .github/workflows/test-and-release.yaml:159]: I think we don't need to expose, do we?

File: .github/workflows/test-and-release.yaml [hunk 2]
```diff
@@ -164,6 +178,8 @@ jobs:
           kubectl wait --for=jsonpath='{.status.phase}'=Running --timeout=300s hazelcast/hazelcast
           echo "Checking Map CR after upgrade"
           kubectl wait --for=jsonpath='{.status.state}'=Success --timeout=300s map/persistent
+          echo "Checking Management Center CR a after upgrade"
```
Comment (kutluhanmetin) [inline .github/workflows/test-and-release.yaml:181]:
```suggestion
          echo "Checking Management Center CR after upgrade"
```

**Review by kutluhanmetin — APPROVED** [review]
Just minor suggestions, LGTM

**Review by SeriyBg — APPROVED** [review]

File: .github/workflows/test-and-release.yaml (hunk 1 above)
Comment (cheels) [reply .github/workflows/test-and-release.yaml:159]: Agree!

**Pull request merged**